appserver/distributions/glassfish-common/src/main/resources/bin/startserv

```diff
@@ -16,10 +17,10 @@
 #
 
 AS_INSTALL=`dirname "$0"`/../glassfish
+AS_INSTALL_LIB="$AS_INSTALL/modules"
 case "`uname`" in
   CYGWIN*) AS_INSTALL=`cygpath --windows $AS_INSTALL`
```

Comment:
Hm... I don't see the need for this block, though, if this line is really needed, it may also be necessary to redefine `AS_INSTALL_LIB` in this case block using `AS_INSTALL` converted to windows path format.

Reply:
Thanks for noticing. I moved `AS_INSTALL_LIB` definitiona under the Cygwin line.

I think not many people use Cygwin to run GlassFish, but it already was in the `lgassfish/bin/asadmin` script (although not in `bin/asadmin`), so I decided to keep support for Cygwin.